assets/wizards/popups/components/segment-group/index.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Segment group component.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { Icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Card, Modal } from '../../../../components/src';
+import SegmentationPreview from '../segmentation-preview';
+import PromptActionCard from '../prompt-action-card';
+import { descriptionForPopup, getCardClassName } from '../../utils';
+
+import {
+	iconInline,
+	iconCenterOverlay,
+	iconTopOverlay,
+	iconBottomOverlay,
+	iconAboveHeader,
+	iconManualPlacement,
+} from './icons';
+import './style.scss';
+
+const addNewURL = ( placement, campaignId, segmentId ) => {
+	const base = '/wp-admin/post-new.php?post_type=newspack_popups_cpt&';
+	const params = [];
+	if ( placement ) {
+		params.push( `placement=${ placement }` );
+	}
+	if ( +campaignId > 0 ) {
+		params.push( `group=${ campaignId }` );
+	}
+	if ( segmentId ) {
+		params.push( `segment=${ segmentId }` );
+	}
+	return base + params.join( '&' );
+};
+
+const SegmentGroup = props => {
+	const { campaignData, campaignId, segment } = props;
+	const [ modalVisible, setModalVisible ] = useState();
+	const { label, id, prompts } = segment;
+
+	let emptySegmentText;
+	if ( 'unassigned' === campaignId ) {
+		emptySegmentText = __( 'No unassigned prompts in this segment.', 'newspack' );
+	} else if ( campaignData ) {
+		emptySegmentText =
+			__( 'No prompts in this segment for', 'newspack' ) + ' ' + campaignData.name + '.';
+	} else {
+		emptySegmentText = __( 'No active prompts in this segment.', 'newspack' );
+	}
+	return (
+		<Card isSmall className="newspack-campaigns__segment-group__card">
+			<h3 className="newspack-campaigns__segment-group__card__segment">
+				{ id ? __( 'Segment: ', 'newspack' ) : '' }
+				{ label }
+				<div>
+					<SegmentationPreview
+						campaignId={ [ campaignId ] }
+						segment={ id }
+						showUnpublished={ true } // Do we need a control for this?
```

Comment:
It might be confusing to see draft prompt along with published ones - it's not _really_ a preview if a segment member will see only published prompts. But that might be a subject of another PR.

Reply:
If I understand correctly, this would need to be a change in `newspack-popups`, which will require @thomasguillot to establish a UI pattern for placing admin-only informational tags on prompts. If true, I think we can file the issue and consider it a task for a new PR.

Reply:
I liked having the control to show or not show unpublished in our last UI iterations, but then I'm not sure how useful it is to publishers. With the control I don't think it's necessary to visually mark the drafts, though, because it becomes an action on the user's part.